component/core/src/main/java/org/exoplatform/social/core/utils/MentionUtils.java

```diff
@@ -18,7 +20,7 @@ public class MentionUtils {
 
     private static final Log LOG                             = ExoLogger.getLogger(MentionUtils.class);
 
-    public static String substituteUsernames(String portalOwner, String message) {
+    public static String substituteUsernames(String portalOwner, String message, ExoSocialActivity activity) {
```

Comment:
Keep method with two parameters available to not add a breaking change for upper addons.

```suggestion
    public static String substituteUsernames(String portalOwner, String message) {
      substituteUsernames(portalOwner, message);
    }

    public static String substituteUsernames(String portalOwner, String message, ExoSocialActivity activity) {
```